etc/kayobe/pulp.yml

```diff
@@ -102,7 +102,7 @@ stackhpc_pulp_deb_repos:
     distribution_name: "ubuntu-cloud-archive-"
     base_path: "ubuntu-cloud-archive/"
     components: "main"
-    distributions: "jammy-updates/{{ openstack_release }}"
+    distributions: "jammy-updates/antelope"
```

Comment:
Again, this shouldn't need to change between releases.

I'd prefer that we annotate things that may need to change.

Reply:
Same as https://github.com/stackhpc/stackhpc-kayobe-config/pull/704#discussion_r1356534594